src/js/footer/changeBannerPadding/index.js

```diff
@@ -1,21 +1,22 @@
 // Adding padding at the top when opening and closing a banner starts here
-  const bannerDisclaimer = document.querySelector('.banner_disclaimer');
-  const pageWrapper = document.querySelector('.page-wrapper');
-
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
@sergei-destinazio is it necessary to have this domContentLoaded event listener?
By right since we are injecting this to header we know that DOM is rendered.
Please let me know if you have any specific reason for this.

Reply:
@ali-hosseini-deriv actually this is not necessary, you are right, I will change the code and create a new pull request.
However, this does not only have the purpose of waiting for the DOM to load. This isolated the function code from the rest of the code in the global space. So I think it's not necessary, but it can be used just in case to avoid unexpected problems in the future.